core/src/main/scala/dagr/core/cmdline/CommandLineException.scala

```diff
@@ -23,63 +23,63 @@
  */
 package dagr.core.cmdline
 
-object CommandLineException {
-  private[cmdline] def format(msg: String, e: Exception): String = {
+private[core] object CommandLineException {
```

Comment:
If the object is `private[core]` do we gain anything by making each exception `private[core]`?  Since no-one can import the object's name or children?  Just thinking that reducing the repetition will make life easier if we change our minds later.